packages/react-native-sdk/src/components/ActiveCall.tsx

```diff
@@ -47,6 +48,8 @@ const InnerActiveCall = (props: ActiveCallProps) => {
   const hasScreenShare = useHasOngoingScreenShare();
   const { callCycleHandlers } = useCallCycleContext();
   const { onHangupCall } = callCycleHandlers;
+
+  useIncallManager({ media: 'video', auto: true });
```

Comment:
We can do that, but then let's say an integrator wants to customize his/her application. They can use the hook and add their own `media` and `auto` values, right? This will make the hook much more usable.

Reply:
The above is the one used by default in the SDK. If I want to change it to `video` and auto `false`, I can do that easily as an integrator.